Network/load-balancing-comparison-matrix.md

```diff
@@ -15,15 +15,18 @@ CenturyLink Cloud offers customers a choice of shared or dedicated load balancin
 |**Feature**   	|**Shared**   	|**Dedicated**
```

Comment:
hopefully its fixed now.  this table is the same used in other KB's just fine and it renders just fine for me in multiple tools.  if it still doesn't show ok would love some help as it matches everything already live and I didn't even change the table code in the original pull request